index.bs

```diff
@@ -546,28 +568,93 @@ canvas.height = Math.max(leftEye.renderHeight, rightEye.renderHeight);
+
+Data returned in calls to {{VRDisplay}}.{{getFrameData()}} using the reference frame's {{VRCoordinateSystem}} will be relative to that fixed orientation and may also include position data if the {{VRDisplay}} performs neck-modeling.
+
+<pre class="idl">
+interface VRAttachedFrameOfReference {
+  readonly attribute VRCoordinateSystem coordinateSystem;
+};
+</pre>
+
+### Attributes ### {#vrattachedframeofreference-attributes}
+
+<dfn attribute for="VRAttachedFrameOfReference">coordinateSystem</dfn>
+This attribute is the {{VRCoordinateSystem}} to be used when relating objects from this body-locked frame of reference to objects in other coordinate systems.
+
+
+## VRStationaryFrameOfReference ## {#interface-vrstationaryframeofreference}
+
+The VRStationaryFrameOfReference interface defines a frame of reference with positional tracking that remains stable relative to the {{VRDisplay}} from which it was created.  The {{VRStationaryFrameOfReference}}'s {{VRCoordinateSystem}} should be used for seated or world-scale experiences.  
+
+Objects rendered using the {{VRCoordinateSystem}} belonging to this frame of reference stay generally in place as the user moves the device around. However, as the user travels large distances and the device adjusts its understanding of the surroundings, objects may drift relative to the world. 
+
+This frame of reference's roll and pitch are relative to gravity while the initial yaw and origin position were based on the {{VRDisplay}} from which the frame of reference was created.
+
+As this frame of reference may become untrackable due to environmental changes, developers are encouraged to be prepared fall back to body-locked content rendered using a {{VRAttachedFrameOfReference}}.
+
+<pre class="idl">
+interface VRStationaryFrameOfReference {
+  readonly attribute VRCoordinateSystem coordinateSystem;
+};
+</pre>
+
+### Attributes ### {#vrstationaryframeofreference-attributes}
+
+<dfn attribute for="VRStationaryFrameOfReference">coordinateSystem</dfn>
+This attribute is the {{VRCoordinateSystem}} to be used when relating objects from this stationary frame of reference to objects in other coodinate systems.
+
+
+## VRStageFrameOfReference ## {#interface-vrstageframeofreference}
+
+The {{VRStageFrameOfReference}} interface represents the stage/play area for devices that support room-scale experiences.  The values within {{VRStageFrameOfReference}} may change after any rendering frame is submitted as the user may re-configure their environment at any time.
+
+Room-scale experiences are expected to use the VRStageFrameOfReference rather than the VRStationaryFrameOfReference, otherwise they may experience drift on world-scale devices.
+
+<pre class="idl">
+interface VRStageFrameOfReference {
+  readonly attribute VRCoordinateSystem coordinateSystem;
 
   readonly attribute float sizeX;
```

Comment:
I was actually wondering about that, especially after seeing Oculus' guardian system. I think it would be a good idea to expose a boundary polygon (defined as 2D points at floor level), but for developer's sanity maybe also provide a simple quad that fits within that as the prescriptive play area.

Worth noting, however, that the boundary polygon could be a hell of a fingerprinting data source.